src/components/AlertBar/AlertBar.vue

```diff
@@ -101,6 +103,9 @@ export default {
       };
     },
   },
+  beforeCreate() {
+    console.warn('Deprecating in favor of BaseBanner.');
```

Comment:
```suggestion
    console.warn('AlertBar is deprecated in favor of BaseBanner.');
```

Just so people know where this warning is coming from.